test/express-slow-down-test.js

```diff
@@ -564,4 +564,34 @@ describe("express-slow-down node module", function () {
         }
       });
   });
+
+  it("should call on onMaxDelayReached function after delay pass limit max delay", function (done) {
+    createAppWith(
+      slowDown({
+        delayAfter: 0,
+        delayMs: 100,
+        maxDelayMs: 200,
+        onMaxDelayReached: function () {
+          done();
+        },
+      })
+    );
+    fastRequest();
+    fastRequest();
```

Comment:
It's been a little while since I worked with these tests, and I think there are some soundness issues with the `fastRequest`/`slowRequest` methods, but I think you at least need to pass in `done` so that it can report errors:

```suggestion
    fastRequest(done);
    fastRequest(done);
```

It might need something else; I'm out of time right now but I'll try and come back to this later.